### PR TITLE
Add ShovanR to the map!

### PR DESCRIPTION
**Pull request by ShovanR**
Adding Shovan and Ottawa to the map!

### CODE REVIEW

File: _pins/ShovanR.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: ShovanR
+latittude: 45.377930
```
Comment (hectorsector) [inline _pins/ShovanR.json]: @ShovanR just check your spelling on the word `latitude`

Comment (bangdamouse) [inline _pins/ShovanR.json]: @ShovanR  you should not have those plus signs in the file. delete them.

**Comment by hectorsector** [comment]
Looks like you're ready to 🚢 @ShovanR! Go ahead and merge when you're ready.

**Comment by davidsharold** [comment]
@ShovanR looks good! 🥇

**Comment by hectorsector** [comment]
@davidsharold you mistakenly closed this PR. Please refrain from closing any PRs that belong to other people so they can get the satisfaction of merging their own 😄

**Pull request merged**